scripts/wrong-filename.sh

```diff
@@ -9,18 +9,20 @@ OUTPUT_FILE="inconsistent-filenames.txt"
 # Remove existing output file (if any)
 rm -f "$OUTPUT_FILE"
 
+IGNORE_LIST=("exclamation mark" "caret")
```

Comment:
Instead of defining the ignore list here, maybe we can move it to a separate file?